addons/website/models/website.py

```diff
@@ -168,6 +169,22 @@ def write(self, values):
         if 'cdn_activated' in values or 'cdn_url' in values or 'cdn_filters' in values:
             # invalidate the caches from static node at compile time
             self.env['ir.qweb'].clear_caches()
+
+        if 'cookies_bar' in values:
+            cookies_view = self.with_context(website_id=self.id).viewref('website.cookie_policy')
+            cookies_page = cookies_view.page_ids.filtered(lambda p: p.website_id.id == self.id)
+            if values['cookies_bar']:
+                if cookies_view and not cookies_page:
+                    self.env['website.page'].create({
+                        'is_published': True,
+                        'is_indexed': False,
```

Comment:
Currently crashes with "no such field", making it harder to test the feature ;-)
```suggestion
                        'website_indexed': False,
```